### PR TITLE
Made album color in trackwin configurable

### LINKED ISSUE #443: Allow theming of album titles in view 1
Right now themes cannot change the color of album titles in the right pane of view 1, they are always bold white text on dark gray. An option to customize this would allow better themes that change the default background color.

### PR DESCRIPTION
**Pull request by jolange**
Closes #443.

### CODE REVIEW

**Comment by mahkoh** [comment]
Please no random whitespace changes.

**Comment by jolange** [comment]
Those are not random, I tried to keep the groups of four lines.

**Comment by mahkoh** [comment]
Then add the new colors at the bottom.

File: options.c [hunk 1]
```diff
@@ -125,7 +128,9 @@ int attrs[NR_ATTRS] = {
 	A_NORMAL,
 	A_NORMAL,
 	A_NORMAL,
-	A_NORMAL
+	A_NORMAL,
```
Comment (jolange) [inline options.c]: While moving new entries to the bottom, I realized that one was acutally missing here.

**Comment by jolange** [comment]
@mahkoh done

**Comment by mahkoh** [comment]
Merged

**Pull request closed without merging**